smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -84,14 +87,33 @@ public IotedgedLinux(string archivePath, Option<RegistryCredentials> credentials
             this.proxy = proxy;
             this.upstreamProtocol = upstreamProtocol;
             this.requireEdgeInstallation = requireEdgeInstallation;
+            this.overwritePackages = overwritePackages;
         }
 
-        public async Task VerifyNotActive()
+        public async Task VerifyNotInstalled()
         {
-            string[] result = await Process.RunAsync("bash", "-c \"systemctl --no-pager show iotedge | grep ActiveState=\"");
-            if (result.First().Split("=").Last() == "active")
+            string[] packages = new string[] { "aziot-edge", "aziot-identity-service" };
```

Comment:
Since we are sharing the same test agent for all of our release branch, I can see an issue when the previous run is using `iotedge, or libiothsm` while the current run is trying to use `aziot-edge, and aziot-identity-service`. We need to find a way to resolve the conflict due to service differences.

Reply:
I'll add iotedge and libiothsm to this list. This function can remove them no matter which branch it's running on, so the test setup always starts with no packages preinstalled.